src/coco/barrel/hover/BarrelHoverCreator.ts

```diff
@@ -6,7 +6,7 @@ import { getBarrelDetailRepresentation } from "../../../common/domain/barrel/uti
 import HoverBuilder from "../../../common/vscode/hover/HoverBuilder";
 import { getConfig } from "../../config/util/configUtil";
 import localization from "../../../localization";
-import { getBarrelAppUrl, getBarrelWebUrl } from "../../../common/domain/openInZeplin/util/zeplinUrls";
+import { getBarrelAppUri, getBarrelWebUri } from "../../../common/domain/openInZeplin/util/zeplinUris";
```

Comment:
```suggestion
import { getBarrelAppUri, getBarrelWebUrl } from "../../../common/domain/openInZeplin/util/zeplinUris";
```

Reply:
Still correct :) but I will revert it for consistency with our other apps.